WordPress/Classes/ViewRelated/Tips/AppTips.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+import SwiftUI
+import TipKit
+import Combine
+
+enum AppTips {
+    static func initialize() {
+        guard Feature.enabled(.tipKit), #available(iOS 17, *) else { return }
+        do {
+            try Tips.configure([.displayFrequency(.daily)])
```

Comment:
If there are many Tips, does this daily frequency configuration mean other tips won't be displayed today if one of them has been displayed?

Reply:
I forgot to remove it after testing – done.
It might be something we go for in the future, but there is no need for it now.